scripts/electronpackage.js

```diff
@@ -0,0 +1,20 @@
+const process = require('process');
+const path = require('path');
+const electronPackager = require('electron-packager');
+
+const baseDir = path.join(__dirname, '..');
+
+electronPackager({
+  dir: baseDir,
+  out: path.join(baseDir, 'build'),
+  icon: path.join(baseDir, 'public/images/icon'),
+  all: process.argv.includes('--all'),
+  asar: true,
+  overwrite: true,
+  ignore: [
+    /^\/(?:[^/]+?\/)*(?:\..+|.+\.less)$/, // dot-files and less files anywhere
+    /^\/(?:\..+|assets|clicktests|coverage|dist|test)\//, // folders in root
```

Comment:
Ignore `scripts`, see #1376.
```suggestion
    /^\/(?:\..+|assets|clicktests|coverage|dist|scripts|test)\//, // folders in root
```